portal-ui/src/screens/Console/Storage/StoragePVCs.tsx

```diff
@@ -118,23 +118,12 @@ const StorageVolumes = ({
       )}
       <h1 className={classes.sectionTitle}>Persistent Volumes Claims</h1>
       <Grid item xs={12} className={classes.actionsTray}>
-        <TextField
+        <SearchBox
```

Comment:
Please include the value prop as it is a new requirement for SearchBox
```suggestion
        <SearchBox
            value={filter}
```